circle.yml

```diff
@@ -30,11 +37,16 @@ deployment:
   release:
     branch: master
     commands:
-      - 'cd $IMPORT_PATH && chmod +x ./build.sh && ./build.sh'
+      - 'cd $IMPORT_PATH && chmod +x ./build.sh && ./build.sh'      
+      - source .env; docker build -t quay.io/turner/harbor-compose:${VERSION} $IMPORT_PATH
       - 'cd $IMPORT_PATH && ghr -t $GITHUB_TOKEN -u $CIRCLE_PROJECT_USERNAME -r $CIRCLE_PROJECT_REPONAME --replace `git describe --tags` dist/'
+      - source .env; docker push quay.io/turner/harbor-compose:${VERSION}
+      - docker push quay.io/turner/harbor-compose:latest
```

Comment:
you are going to have to tag latest first in the build before you push it, or this will fail. 

```
 source .env; docker push quay.io/turner/harbor-compose:${VERSION}
 source .env; docker tag quay.io/turner/harbor-compose:${VERSION} quay.io/turner/harbor-compose:latest
 docker push quay.io/turner/harbor-compose:latest
```